### DISCUSSION TITLE
How/where to enter connection data

### DISCUSSION THREAD

**Discussion by NikolayDA** [greghesp/ha-bambulab#94] [Q&A] [answered]
Hi all,

I installed HACS and the ha-bambulab as shown in the readme without any issues. But now I don't know where to enter the data to actually connect to the printer. Do I have to add the information to this file '/config/custom_components/bambu_lab/strings.json'?

Kind regards,

Noob Nikolay

**Comment by npike** [comment, marked as answer]
Go to Settings -> Integrations. Add an integration, search for Bambu Lab

![Screenshot_20230414-142848](https://user-images.githubusercontent.com/144590/232158190-6a8ca9d8-7889-47e9-8df9-756551283686.png)

**Reply by NikolayDA** [reply]
Thank you npike! You make the internet a great place.
